app/components/ProjectRevision/ChangeReasonWidget.tsx

```diff
@@ -0,0 +1,117 @@
+import { useUpdateProjectRevision } from "mutations/ProjectRevision/updateProjectRevision";
+import { Button, Textarea } from "@button-inc/bcgov-theme";
+import ReadOnlyWidget from "lib/theme/widgets/ReadOnlyWidget";
+import { WidgetProps } from "@rjsf/core";
+import { useState } from "react";
+import { graphql, useFragment } from "react-relay";
+
+const ChangeReasonWidgetFragment = graphql`
+  fragment ChangeReasonWidget_projectRevision on ProjectRevision {
+    id
+    changeStatus
+  }
+`;
+
+// Custom widget to update the revision change reason
```

Comment:
I agree that the naming is confusing, I had the same issue before in other fields since not all fields are named the same as their representative in the DB. I usually first look at the schema to understand the relationship between those.

Reply:
Sounds like you already put a lot of thought into it, happy to leave it